### ISSUE TITLE
'minDate/maxDate' datepicker options not set correctly on form

### ISSUE THREAD

**Issue by Flynsarmy** [octobercms/october#909]
If I have a form field like so:
```yaml
published_at:
  label: Published on
  type: datepicker
  mode: datetime
  minDate: 1900-01-01
  maxDate: 2020-01-01
```
The datepicker element on the form will show the year as 1970 despite it being 2015 in the database. This is caused by the YAML file converting 1900-01-01 to -2208988800 and 2020-01-01 to 1577836800 rather than simply leaving them as strings:
```html
<!-- Date -->
<div
    id="DatePicker-formPublishedAt-published_at"
    class="field-datepicker"
    data-control="datepicker"
    data-min-date="-2208988800"
    data-max-date="1577836800">
    <input
        type="text"
        id="DatePicker-formPublishedAt-input-published_at"
        name="Post[published_at]"
        value="2015-01-24"
        class="form-control align-right"
        autocomplete="off">
</div>
```

I've whipped up a demo repo complete with steps to replicate [here](https://github.com/Flynsarmy/oc-baddate-plugin.git).

**Comment by karnold** [comment]
I am also experiencing this by manually invoking the datepicker plugin as well.

despite initiating the picker with the following

```
                    <div id="DatePicker-birthdate" 
                        class="field-datepicker" 
                        data-control="datepicker" 
                        data-min-date="1900-01-01" 
                        data-max-date="2020-12-31"
                    >
```

the year selection will only show from 2004 to 2020

**Comment by psoots** [comment]
Temporary workaround is to quote the date string in the YAML file:

```yaml
published_at:
  label: Published on
  type: datepicker
  mode: datetime
  minDate: "1900-01-01"
  maxDate: "2020-01-01"
```

**Comment by daftspunk** [comment]
Fixed by #1475
